master/pkg/searcher/custom_search.go

```diff
@@ -1,147 +0,0 @@
-package searcher
-
-import (
-	"encoding/json"
-
-	"github.com/pkg/errors"
-	"google.golang.org/protobuf/types/known/structpb"
-
-	"github.com/determined-ai/determined/master/pkg/model"
-	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
-	"github.com/determined-ai/determined/proto/pkg/experimentv1"
-)
-
-type (
-	customSearchState struct {
-		SearchMethodType     SearchMethodType `json:"search_method_type"`
-		SearcherEventQueue   *SearcherEventQueue
-		CustomSearchProgress float64
-	}
-
-	customSearch struct {
-		expconf.CustomConfig
-		customSearchState
-	}
-)
-
-func newCustomSearch(config expconf.CustomConfig) SearchMethod {
```

Comment:
I believe the `expconf.CustomConfig` type still exists, do we want to remove this as well?

Reply:
This type only seems to be used [here](https://github.com/determined-ai/determined/blob/remove-custom-searcher/master/internal/db/postgres_experiments_intg_test.go#L759) in our experiment integration tests

Reply:
good point, removed it and the calling piece in that intg test